sympy/functions/special/error_functions.py

```diff
@@ -1399,6 +1409,12 @@ class li(Function):
     1/log(z)
 
     Defining the ``li`` function via an integral:
+    >>> from sympy import integrate
```

Comment:
I think you need an empty space above the example lines or it won't render correctly.

Reply:
Yes, added it.